service/src/test/java/bio/terra/policy/service/policy/PolicyGroupConstraintTest.java

```diff
@@ -77,11 +76,10 @@ void groupConstraintTest_combineEmptyDestination() throws Exception {
   void groupConstraintTest_combineEmptySourceAndDestination() throws Exception {
     var groupConstraint = new PolicyGroupConstraint();
 
-    // Only adding region constraints.
     var dependentPolicy =
-        new PolicyInput(TERRA_NAMESPACE, REGION_CONSTRAINT, buildMultimap(REGION_KEY, GROUP_NAME));
+        new PolicyInput(TERRA_NAMESPACE, GROUP_CONSTRAINT, buildMultimap(REGION_KEY));
     var sourcePolicy =
-        new PolicyInput(TERRA_NAMESPACE, REGION_CONSTRAINT, buildMultimap(REGION_KEY, GROUP_NAME));
+        new PolicyInput(TERRA_NAMESPACE, GROUP_CONSTRAINT, buildMultimap(REGION_KEY));
```

Comment:
this was actually an invalid test as written, using a group constraint policy to combine region constraint policies